packages/flutter_test/test_fixes/flutter_test/animation_sheet_builder.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('test', (WidgetTester tester) async {
```

Comment:
Can you add the same 'Changes made in...' comment for each?

This helps us easily match the fix to the test

Reply:
Added!